cfgov/v1/tests/test_social_sharing_image_validation.py

```diff
@@ -0,0 +1,57 @@
+from django.core.exceptions import ValidationError
+from django.test import TestCase
+
+from model_mommy import mommy
+
+from v1.models.browse_filterable_page import BrowseFilterablePage
+from v1.models.images import CFGOVImage
+from v1.tests.wagtail_pages.helpers import save_new_page
+
+
+class TestSocialSharingImage(TestCase):
+    def setUp(self):
+        self.page = BrowseFilterablePage(
+            title='Browse Filterable Page',
+            slug='browse-filterable-page',
+        )
+        save_new_page(self.page)
+
+    def test_validation_error_thrown_if_width_height_too_large(self):
+        self.page.social_sharing_image = mommy.prepare(CFGOVImage, width=5000, height=5000)
+        with self.assertRaisesRegexp(
+            ValidationError,
+            'Social sharing image must be less than 4096w x 4096h'
+        ):
+            self.page.save()
+
+    def test_validation_error_thrown_if_width_too_large(self):
+        self.page.social_sharing_image = mommy.prepare(CFGOVImage, width=4097, height=1500)
+        with self.assertRaisesRegexp(
+            ValidationError,
+            'Social sharing image must be less than 4096w x 4096h'
+        ):
+            self.page.save()
+
+    def test_validation_error_thrown_if_height_too_large(self):
+        self.page.social_sharing_image = mommy.prepare(CFGOVImage, width=1500, height=4097)
+        with self.assertRaisesRegexp(
+            ValidationError,
+            'Social sharing image must be less than 4096w x 4096h'
+        ):
+            self.page.save()
+
+    def test_validation_error_thrown_if_width_too_small(self):
+        self.page.social_sharing_image = mommy.prepare(CFGOVImage, width=1199, height=1500)
+        with self.assertRaisesRegexp(
+            ValidationError,
+            'Social sharing image must be greater than 1200w x 630h'
+        ):
+            self.page.save()
+
+    def test_validation_error_thrown_if_height_too_small(self):
+        self.page.social_sharing_image = mommy.prepare(CFGOVImage, width=1500, height=629)
+        with self.assertRaisesRegexp(
+            ValidationError,
+            'Social sharing image must be greater than 1200w x 630h'
```

Comment:
This (and the above) message will also need to change here in the test according to the change made to the actual message.